include/Song.h

```diff
@@ -97,6 +97,7 @@ class EXPORT Song : public TrackContainer
 
 
 	void processNextBuffer();
+	void processAutomations(const TrackList& tracks, MidiTime timeStart, fpp_t frames, int tcoNum);
```

Comment:
I'm going to add a unit test for this feature, so I'll leave it public. We can gladly refactor to split this into a public and a private part some day.